default_app/main.js

```diff
@@ -255,7 +255,7 @@ function loadApplicationPackage (packagePath) {
       app.focus()
       dialog.showErrorBox(
         'Error opening app',
-        'The app provided is not a valid Electron app, please read the docs on how to write one:\n' +
+        'Unable to open or find an Electron app. The docs on how to write an app can be found here:\n' +
```

Comment:
I think this is an improvement.

URLs in dialogs are less than ideal since they can't be clicked or copied.

What do you think about adding another button in the dialog that says "View Guide" or "View Docs" or "Learn More" or something and when clicked opens the docs? And then we can remove the URL completely from the message.